tests/templates/kuttl/external-access/50-assert.yaml.j2

```
@@ -0,0 +1,24 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+metadata:
+  name: test-log-endpoint
+timeout: 240
+commands:
+{% if test_scenario['values']['executor'] == 'celery' %}
+  - script: |
+      set -eu
+
+      # Log-Endpoint Test:
+      # This is executed from the Webserver as JWT keys must be present.
+      # A small server is started on each worker that serves the logs on its
+      # 8793 port for the Webserver: we don't use the token as that is an
+      # internal implementation, but check that the end-point is reachable,
```

optional: You know English way better than me. However, in all the k8s docs I have only seen endpoint so far, do you have any knowledge on this? :)
```suggestion
      # internal implementation, but check that the endpoint is reachable,
```

---

I think you're right :)